.env.example

```
@@ -1,5 +1,15 @@
 NODE_ENV=development
-APP_KEY=73rHnSEgVPLfFLpTk8omwzBrMk-c75rX
-DRIVE_DISK=local
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_USER=root
+DB_PASSWORD=root
+DB_DATABASE=corelab_challenge
+
+FRONTENT=http://localhost:3000
+DRIVE_DISK=fs
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)